content/docs/rfcs/37/README.md

```diff
@@ -0,0 +1,151 @@
+depending if `Hash(session-id)` is sent on `/wakunoise/1/<ct-id>/proto` to the other party in encrypted form or not, respectively. 
+
+When a Noise session is publicly marked as stale, 
+network peers MAY discard all [stored](https://rfc.vac.dev/spec/13/) messages addressed to the content topic `/wakunoise/1/<ct-id>/proto`. 
+In this the case and in order for parties to retrieve any eventually delayed message, 
+peers SHOULD wait a fixed amount of time before discarding stored messages corresponding to a stale Noise session. 
+
+A stale Noise session cannot be directly marked as active
+and parties are required to instantiate a new Noise session if they wish to communicate again. 
+
+However, parties can optionally include the `session-id` corresponding to a stale Noise session in the [prologue information](https://noiseprotocol.org/noise.html#prologue) employed in the Noise handshake they execute to instantiate their new Noise session. 
+This effectively emulates a mechanism to _"re-activate"_ a stale Noise session by binding it to a newly created active Noise session.
+
+In order to reduce users' metadata leakage, it is desirable (as suggested in [35/WAKU2-NOISE](https://rfc.vac.dev/spec/35/#after-handshake)) that content topics used for communications change every time a new message is exchanged.
+This can be easily realized by employing a key derivation function to compute a new `session-id` from the previously employed one (e.g. `session-id = HKDF(prev-session-id)`),
+while keeping the Inbound/outbound Cipher States, the content topic derivation mechanism and the stale mechanism the same as above.
+In this case, when one party sends **and** receives at least one message, 
+he SHALL publicly mark as stale all Noise sessions relative to messages exchanged before the earlier of these two send/receive events.
+
+# Multi-Device support
+
+Alice and Bob might possess one or more devices (e.g. laptops, smartphones, etc.) they wish to use to communicate.
+In the following we assume Alice and Bob to possess $N$ and $M$ devices, respectively.
+
+Since a Noise session contains cryptographic material required to encrypt and decrypt messages exchanged on a pre-defined content topic derived from a `session-id`, 
+messages should be encrypted and decrypted within the Noise session instantiated between the currently-in-use sender's and receiver's device.
+
+This is achieved through two main supported session management mechanisms that we called `N11M` and `NM`, respectively.
+
+## The $N11M$ session management mechanism
+
+In a $N11M$ setting, each party's device shares the same Noise session information used to encrypt and decrypt messages exchanged with the other party.
+
+![](./N11M.png)
+
+More precisely, once the first Noise session between any of Alice's and Bob's device is instantiated, 
+its session information is securely propagated to all other devices, 
```

Comment:
Are other devices of the same person disconnected from the network? is only one device the entry point of other devices?

Reply:
Once Noise sessions are instantiated between devices, they can go all offline, since sync information would be propagated on the agreed content topic and made available by store nodes. It is one device at a time that communicates with the other party. If multiple devices (with same session information) send a message simultaneously, then the first received message will be the only decrypted correctly, while the others will have an invalid nonce for the receiver. Hence, based on the session information propagated and how Noise encryption works, only one device per message can communicate to the other party.

Reply:
> one device per message can communicate to the other party.

This is not clear from the description, please clarify it.

Reply:
Addressed in https://github.com/vacp2p/rfc/commit/f4c6b40ed3647120770a6330355c09e854d45e0c